### PR TITLE
Add param to only return the value of an aggregate by its key

### PR DESCRIPTION
**Pull request by aliel**

### CODE REVIEW

File: src/aleph/web/controllers/aggregates.py [hunk 1]
```diff
@@ -62,17 +60,27 @@ async def address_aggregate(request: web.Request) -> web.Response:
...
     if not aggregates:
         raise web.HTTPNotFound(text="No aggregate found for this address")
 
+    if query_params.only_value and query_params.keys and len(query_params.keys) == 1:
+        output = {}
+        target_key = query_params.keys[0]
+        for result in aggregates:
+            output[result[0]] = result[1]
+
+        return web.json_response(output[target_key])
```
Comment (hoh) [inline src/aleph/web/controllers/aggregates.py:77]: Can you add a test to cover this case ?

File: src/aleph/web/controllers/aggregates.py [hunk 2]
```diff
@@ -22,6 +19,7 @@ class AggregatesQueryParams(BaseModel):
     keys: Optional[List[str]] = None
     limit: int = DEFAULT_LIMIT
     with_info: bool = False
+    only_value: bool = False
```
Comment (hoh) [inline src/aleph/web/controllers/aggregates.py:22]: Nitpick: would `value_only` make more sense than `only_value` ? Seems more like English literal :thinking:

File: src/aleph/web/controllers/aggregates.py [hunk 3]
```diff
@@ -62,17 +60,27 @@ async def address_aggregate(request: web.Request) -> web.Response:
...
-                session=session, owner=address, with_info=query_params.with_info
+                session=session, owner=address,
+                with_info=query_params.with_info, keys=query_params.keys
             )
         )
 
     if not aggregates:
         raise web.HTTPNotFound(text="No aggregate found for this address")
 
+    if query_params.value_only and query_params.keys and len(query_params.keys) == 1:
```
Comment (hoh) [inline src/aleph/web/controllers/aggregates.py:71]: What happens else ? For example if there are multiple keys ?

File: src/aleph/web/controllers/aggregates.py [hunk 4]
```diff
@@ -62,17 +60,27 @@ async def address_aggregate(request: web.Request) -> web.Response:
...
+        output = {}
+        target_key = query_params.keys[0]
+        for result in aggregates:
+            output[result[0]] = result[1]
```
Comment (hoh) [inline src/aleph/web/controllers/aggregates.py:75]: Why not return it here ?

File: src/aleph/web/controllers/aggregates.py (hunk 3 above)
Comment (MHHukiewitz) [reply src/aleph/web/controllers/aggregates.py:71]: It will continue as normal

File: src/aleph/web/controllers/aggregates.py (hunk 4 above)
Comment (MHHukiewitz) [reply src/aleph/web/controllers/aggregates.py:75]: There might be multiple but we are only interested in the target_key

**Review by MHHukiewitz — APPROVED** [review]
LGTM

**Pull request merged**